### DISCUSSION TITLE
Hovercard render

### DISCUSSION THREAD

**Discussion by ivanov-v** [ariakit/ariakit#1133] [Questions] [answered]
Hello. Why is the Hovercard not being rendered in the portal? Is such an option planned? 
In my project there are such styles and they affect the card:
![image](https://user-images.githubusercontent.com/13759065/159519131-1d493b20-a419-44b4-96fa-0cc39753cd4d.png)

**Comment by diegohaz** [comment, marked as answer]
`<Hovercard portal>`

**Reply by ivanov-v** [reply]
Nice! Why not by default?

**Reply by diegohaz** [reply]
Because it's simpler without a portal and it works for most cases.
